### DISCUSSION TITLE
Data추출이 막힌건지요?

### DISCUSSION THREAD

**Discussion by won5450** [changdoc/naver-vaccine-macro#280] [Q&A]
네이버에서 잔여백신 사이트 아웃룩을 약간 변경한 이후로 (8월 25일) 병원목록 data추출이 안되고 있습니다.
저만 안되는건지... 확인 부탁 드리겠습니다. 감사 합니다.

**Comment by changdoc** [comment]
https://github.com/changdoc/naver-vaccine-macro/issues/278
이 이슈 내용 보시면 파악이 가능한데요. 네이버 지도 쪽 동작이 좀 바뀌어 데이터 추출 하는 부분에 변경이 있습니다.
가이드 새로 작성해서 공유드리겠습니다.

**Comment by changdoc** [comment]
우선 사용하시려면 최신 코드로 설치를 하시고
데이터는 아래 스크린샷을 참조하시면 추출 가능하실 듯 합니다.
<img width="1934" alt="스크린샷 2021-08-26 오후 5 45 53" src="https://user-images.githubusercontent.com/1879494/130931906-bb940652-fc16-4b91-bf89-d50dc8d9fe02.png">
